docs/content/concepts/partitions-schedules-sensors/sensors.mdx

```diff
@@ -183,6 +183,62 @@ def asset_a_and_b_sensor_with_skip_reason(context):
         )
 ```
 
+### Asset reconciliation sensor <Experimental />
+
+Dagster provides an out-of-box sensor that will monitor a provided set of assets and automatically materialize the assets if their upstream dependencies are materialized. You can create this sensor using the <PyObject object="build_asset_reconciliation_sensor"/> function. You can specify if the sensor should materialize an asset when **any** of its parent assets have been materialized or after **all** of its parent assets have been materialized. This sensor can help you keep your assets up-to-date without needing to write custom schedules or sensors for each asset. To use an asset reconciliation sensor, add it to your repository:
+
+```python file=/concepts/partitions_schedules_sensors/sensors/asset_reconciliation_sensor.py startafter=start_asset_reconciliation_sensor endbefore=end_asset_reconciliation_sensor
+@repository
+def repository_1():
+    return [
+        asset_a,
+        asset_b,
+        asset_c,
+        build_asset_reconciliation_sensor(
+            asset_selection=AssetSelection.assets(asset_b, asset_c),
+            name="asset_reconciliation_sensor",
+        ),
+    ]
+```
+
+This sensor will materialize `asset_b` whenever any of its parents materialize, and it will materialize `asset_c` whenever any of its parents materialize.
```

Comment:
I want to avoid implying that the sensor will update `asset_b` and `asset_c` even if only `asset_b`'s parents have updated, but that leaves me with a bit of wordy/repetitive sentence. It reads ok to me, but if you get tripped up on it, i'll start tweaking it some more